src/FunderMaps.Core/Threading/DispatchManager.cs

```diff
@@ -54,10 +54,8 @@ public DispatchManager(IOptions<BackgroundWorkOptions> options, ILogger<Dispatch
             _options = options?.Value ?? throw new ArgumentNullException(nameof(options));
             _serviceScopeFactory = serviceScopeFactory ?? throw new ArgumentNullException(nameof(serviceScopeFactory));
 
-            // NOTE: Add one more worker than configured. This will set the lower bound to 1
-            //       and will misalign the number of workers-to-CPU-core ratio. Doing so will
-            //       always keep a single core free for other processes.
-            workerPoolHandle = new(_options.MaxWorkers + 1, (_options.MaxWorkers * 2) + 1);
+            // NOTE: Allocate the worker requested and reserve twice as much.
```

Comment:
Why reserve twice as much?

Reply:
Because it can be beneficial on large servers to allow for overhead. However thats not the case anymore